src/Kernel/ClassOrganization.class.st

```diff
@@ -240,12 +217,9 @@ ClassOrganization >> protocolNameOfElement: aSelector [
 { #category : #accessing }
 ClassOrganization >> protocolNameOfElement: aSelector ifAbsent: aBlock [
 
-	^ (self protocolsOfSelector: aSelector)
-		  ifEmpty: [
-			  (organizedClass includesSelector: aSelector)
-				  ifTrue: [ Protocol unclassified ]
-				  ifFalse: [ aBlock value ] ]
```

Comment:
Here I removed this hack because I checked in a running image and I have no selector that is in the class but not in a protocol and I think we should not allow that state to happen